src/main/scanLogic/scanRunners/sastScan.ts

```diff
@@ -92,6 +94,7 @@ export class SastRunner extends JasRunner {
             type: this._scanType,
             roots: this._config.GetSourceRoots(this._scanType),
             language: this._config.GetScanLanguage(),
+            user_rules: Configuration.getSastCustomRulesPath(this._logManager),
```

Comment:
I see `user_rules` is assigned here - but where is it used? who calls this param inside `Configuration` object?

Reply:
@hadarshjfrog, The configuration value is coming from the settings in the vscode as shared at the details
![image](https://github.com/jfrog/jfrog-vscode-extension/assets/49212512/eac2e40e-735c-420f-998e-4e07a230078b)

The value is than fetched using `Configuration.getSastCustomRulesPath` and we pass it to the SAST scanner that uses it while running.